src/systems/filecoin_markets/storage_market/storage_market_actor.id

```diff
@@ -2,18 +2,21 @@ import actor "github.com/filecoin-project/specs/systems/filecoin_vm/actor"
 import block "github.com/filecoin-project/specs/systems/filecoin_blockchain/struct/block"
 import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address"
 import deal "github.com/filecoin-project/specs/systems/filecoin_markets/deal"
+import sector "github.com/filecoin-project/specs/systems/filecoin_mining/sector"
 
 type StorageParticipantBalance struct {
     Locked     actor.TokenAmount
     Available  actor.TokenAmount
 }
 
 type BalancesHAMT {addr.Address: StorageParticipantBalance}
-type DealsAMT {deal.DealID: deal.StorageDeal}
+type PublishedDealsAMT {deal.DealID: deal.StorageDeal}
+type ActiveDealsAMT {deal.DealID: deal.ActiveStorageDeal}
 
 type StorageMarketActorState struct {
-    Balances  BalancesHAMT
-    Deals     DealsAMT
+    Balances        BalancesHAMT
+    PublishedDeals  PublishedDealsAMT
+    ActiveDeals     ActiveDealsAMT
```

Comment:
distinction between Published and Active deals not super clear, are these just two different states of the same thing? maybe say so. it looks though than an "ActiveDeal" includes other stuff (some accounting/tallies)

Reply:
Yes, they are two states of a deal, PublishedDeals are created when published but they only become active after `ProveCommit`. `ActiveDeal` also keeps track of all the accounting information related to payment and collateral that are not needed for PublishedDeals. Also not all PublishedDeals will make it into ActiveDeals (hence we saw delete and abort - will fix that one too)